src/main/resources/com/uber/jenkins/phabricator/HarbormasterConduitNotifier/config.jelly

```diff
@@ -0,0 +1,6 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <f:entry title="Build Target" field="buildTarget"
+           description="Name of the env variable containering the Build Target PHID">
```

Comment:
It's the name of the the paramter that will have build target phid.  I'm cool with calling it `buildTarget` `buildTargetPHID` or whatever sounds the least confusing.